crates/astria-sequencer-relayer/tests/blackbox/main.rs

```diff
@@ -246,12 +260,13 @@ async fn block_from_other_proposer_is_skipped() {
                 break;
             }
         }
+        sequencer_relayer.shutdown();
```

Comment:
Why is the shutdown happening here? Because the object is moved into the block?

Yeah, that's ugly. It's not the best fix, but can you just hand back the object instead and call `shutdown` outside? That's marginally better. I mean this:

```rust
let observe_blobs = async move {
    // todo!("do things with sequencer_relayer")
   (sequencer_relayer, observe_blobs)
};

Then you can get the object back into the outer scope.

Reply:
removed in favour of `Drop`